build.gradle.kts

```diff
@@ -387,7 +390,9 @@ tasks {
     }
 
     build {
+        dependsOn("downloadAndUnpackHelmDocs")
```

Comment:
Do you need this dependency? The same dependency is also required by the `buildHelmChartReadme` task...

Reply:
I wanted to have the downloads happening before the build. Otherwise at the end of the build we do another download, which works, but is a bit unexpected.